pkg/api/postage.go

```diff
@@ -295,17 +297,22 @@ func (s *Service) postageGetStampHandler(w http.ResponseWriter, r *http.Request)
 	}
 	id, err := hex.DecodeString(idStr)
 	if err != nil {
-		s.logger.Debug("get stamp issuer: decode batch id string failed", "string", idStr, "error", err)
-		s.logger.Error(nil, "get stamp issuer: decode batch id string failed")
-		jsonhttp.BadRequest(w, "invalid batchID")
+		logger.Debug("decode batch id string failed", "string", idStr, "error", err)
+		logger.Error(nil, "decode batch id string failed")
+		jsonhttp.BadRequest(w, "cannot find batch")
```

Comment:
This message was supposed to be renamed. The below message is correct.

Reply:
Fixed.